constraints/miden-vm/range-checker.air

```diff
@@ -0,0 +1,32 @@
+mod RangeCheckerAir
+
+fn binary_not(e: scalar) -> scalar:
+    return 1 - e
+
+fn change(e: scalar) -> scalar:
+    return e' - e
+
+ev check_binary(main: [v]):
+    enf v^2 - v = 0
+
+ev range_checker(main: [t, s0, s1, v]):
+    # Check selector flags are binary
+    # t^2 - t = 0
+    # s0^2 - s0 = 0
+    # s1^2 - s1 = 0
+    let selectors = [t, s0, s1]
+    enf check_binary([s]) for s in selectors
+
+    # Constrain the row transitions in the 8-bit section of the table
+    enf change(v) * (change(v) - 1) = 0 when !t0'
+
+    # Constrain the transition from 8-bit to 16-bit section of the table
```

Comment:
If I got it right, this is just a "section separator" comment that is telling us what constraints are located below. For me it's a little bit confusing -- at first I thought that constraint is just missing. Maybe we should somehow highlight such  comments? For example, using two `#`s instead of one, and, maybe, using in before and after comment? Something like `## Comment ##`. Wdyt?

Reply:
I moved the constraints to another evaluator function instead. I'm wondering if as a general rule maybe we can try to move a group of constraints into another evaluator (which is also what you've done in the other chiplet constraints PRs)?

Reply:
Yes, I think it is a good idea. It looks better in my opinion.

Reply:
> I'm wondering if as a general rule maybe we can try to move a group of constraints into another evaluator (which is also what you've done in the other chiplet constraints PRs)?

Yes, I prefer this as a general rule too